scss/footer.scss

```diff
@@ -0,0 +1,19 @@
+.train-footer{
+  background-color: $training-blue; 
+  display: flex;
+  flex-direction: column; 
+  padding: 40px 20px 20px 20px;  
+
+  .txt {
```

Comment:
I'd try to avoid nesting attributes by using more descriptive classes like `footer-text` or `footer-slogan`. keeping things with the least nesting possible is very important to  easily override them if necessary